resources/design-document.md

```diff
@@ -128,20 +128,24 @@ String contestStatus;
       `GameNotFoundException`
```

Comment:
For consistency with the other endpoints (and restful api naming conventions), what do you think about ``/game/:uniqueId/games/:gameName` as the path for this "Get Game Endpoint"?